File: src/main/java/net/morbz/minecraft/blocks/StandingSignBlock.java
package net.morbz.minecraft.blocks;

public class StandingSignBlock implements IBlock {

    @Override
    public byte getBlockId() {
        return (byte)Material.STANDING_SIGN.getValue();
    }

    @Override
    public byte getBlockData() {
        return 0;
    }

    @Override
    public int getTransparency() {
        return Material.STANDING_SIGN.getTransparency();
    }
}

File: src/main/kotlin/org/matthelliwell/minecraft/blocks/StandingSignBlock.kt
package org.matthelliwell.minecraft.blocks

class StandingSignBlock : IBlock {

    override val blockId: Byte
        get() = Material.STANDING_SIGN.value.toByte()

    override val blockData: Byte
        get() = 0

    override val transparency: Int
        get() = Material.STANDING_SIGN.transparency
}
